MVPSt_2023_Feb_Competition/MVPSt_2023_Feb_Competition/Utilities/CommonDriver.cs

```diff
@@ -0,0 +1,181 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO.Enumeration;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using MVPSt_2023_Feb_Competition.Pages;
+using NUnit.Framework;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Chrome;
+using OpenQA.Selenium.Support.UI;
+using MVPSt_2023_Feb_Competition.Utilities;
+using AventStack.ExtentReports.Reporter;
+using AventStack.ExtentReports;
+using System.IO;
+
+namespace MVPSt_2023_Feb_Competition.Utilities
+{
+    [TestFixture]
+    public class CommonDriver
+    {
+        // for Extent Reports
+        public static ExtentReports extentReportObj = null;
+        public static ExtentTest test = null;
+
+        // Sign Out flag used by SignOut method to determine whether or not to attempt to SignOut
+        private static bool signoutBool = false;
+
+        public static IWebDriver driver;
+
+        public static string filePath { get; set; }
+        
+        // variables from configuration file
+        public static string filePath2 { get; set; }
+        public static string sheetName2 { get; set; }
+        public static string sheetName2Row { get; set; }
+        public static string filePath3 { get; set; }
+        public static string sheetName3 { get; set; }
+        public static string sheetName3Row { get; set; }
+
+        // converted variables from string to integer
+        public static int sheetName2RowInt { get; set; }
+        public static int sheetName3RowInt { get; set; }
+
+        //// for Extent Reports
+        //public static ExtentTest test;
+        //public static ExtentReports extent;
+
+        [SetUp]
+        public static void SetupActions()
+        {
+            Console.WriteLine("**Starting 'Trade Your Skills' portal script from CommonDriver");
+
+            // create the Extent Report
```

Comment:
again break down into readable methods

Reply:
Please clarify reason for breaking down the SetupActions method of the CommonDriver's non-repeatable code into separate methods. I have attempted to explain each portion of code in the SetupActions method with a clear comment. Are the comments unclear or should the portions of code be grouped in a different sequence?

Reply:
Resolving conversation as mentor approved submission without requiring this update to be completed.